--- a/app/listeners/webhook_listener.rb
+++ b/app/listeners/webhook_listener.rb
@@ -3,6 +3,9 @@
     conversation = extract_conversation_and_account(event)[0]
     changed_attributes = extract_changed_attributes(event)
     inbox = conversation.inbox
+
+    Crm::EnsureFromConversationJob.perform_later(conversation.id, conversation.account_id)
+
     payload = conversation.webhook_data.merge(event: __method__.to_s, changed_attributes: changed_attributes)
     deliver_webhook_payloads(payload, inbox)
   end
